setup.py

```diff
@@ -10,7 +10,7 @@
     author_email="info@zyte.com",
     url="https://github.com/scrapy-plugins/scrapy-zyte-api",
     packages=["scrapy_zyte_api"],
-    install_requires=["zyte-api>=0.1.2"],
+    install_requires=["zyte-api>=0.1.2", "scrapy>=2.6.0"],
```

Comment:
Given https://github.com/pypa/pip/pull/4500 is there, it could be ok to have scrapy >= 2.6.0 in install_requires.
On the other hand, let's say I'm using scrapy 2.4, and want to use Zyte API. I install scrapy-zyte-api, but suddenly Scrapy gets upgraded, probably breaking things in the project.

What do you think about failing at runtime (or at import time?), instead of using install_requires? For the scenario above it seems like a better behavior: you get a clear message that Scrapy 2.6+ is required, and you can decide on how you're going to approach it.

Reply:
> Also, in the scenario where users have an older Scrapy and there is a version of scrapy-zyte-api that supports that version, things would just work™ with a requirements.txt with Scrapy<x.y.z and scrapy-zyte-api. With the run-time approach, users would need to manually find out which version of scrapy-zyte-api to install.

That's nice, I haven't considered it. It looks like my packaging knowledge is pretty outdated :) Thanks @Gallaecio!